resources/views/movies/index.blade.php

```diff
@@ -0,0 +1,20 @@
+<x-app-layout>
```

Comment:
As I mentioned before this might throw an error for unauthenticated users. So we might need to modify the navigation bar a bit to accommodate guests. Or alternatively, you can try to use the guest layout for this one?

Reply:
Mangage to get the naviagation bar to work for index and show, showing in the top right corner either login/register if the user is a guest or the user name if the user is currently logged in